03-rnaseq/clustering_rnaseq_01_heatmap.Rmd

```diff
@@ -218,7 +218,8 @@ metadata <- readr::read_tsv(metadata_file)
 
 # Read in data TSV file
 expression_df <- readr::read_tsv(data_file) %>%
-  # Here we are going to store the gene IDs as rownames so that we can have a numeric matrix to perform calculations on later
+  # Here we are going to store the gene IDs as rownames so that
```

Comment:
```suggestion
  # Here we are going to store the gene IDs as row names so that
```

Reply:
This is a style choice that I left because it is this way in many other places in the repo... we could change them all, but I don't want to change it piecemeal.

Reply:
I've been trying to change them whenever I see them, so I think its inconsistent at best. But I can file a separate search and replace type PR.

Reply:
I can add in most of them here.